### DISCUSSION TITLE
styleDark parameter not working in Safari

### DISCUSSION THREAD

**Discussion by CutlerSheridan** [add2cal/add-to-calendar-button#473] [Q&A] [answered]
I have a button with the parameters `lightMode='dark'` and `styleDark='--btn-shadow:none'`.  In Chrome, this correctly applies the dark style to the button and removes the shadow.  In Safari (both on iOS and macOS), it applies the dark style, but `styleDark` does not change anything (and this is true no matter what style I try to change, not just the box-shadow).

Oddly, if `lightMode='light'`,  then `styleLight` DOES work as expected in Safari.

Is this a known issue?

**Comment by jekuer** [comment]
Have an idea where this is coming from.
We have a workaround in place for safari. I guess this is not properly applied to the override stuff.
Will check asap.

**Reply by CutlerSheridan** [reply]
Awesome, appreciate you checking either way!

**Reply by jekuer** [reply]
Please excuse the delay. Fix is prepared to be included in the next release, which should be rolled out over the next 1-2 weeks.

**Reply by jekuer** [reply, marked as answer]
Took some more time, but it is now working in the latest version (2.5)! 😃

**Reply by CutlerSheridan** [reply]
Excellent, thanks for keeping this great package updated :)
